src/config/constants/constants.ts

```diff
@@ -43,6 +43,7 @@ export const DEFAULT_LOCALE = 'en-US';
 export const ITEMS_TABLE_ROW_ICON_COLOR = '#333333';
 
 export const USERNAME_MAX_LENGTH = 30;
+export const HEADER_ITEM_NAME_MAX_LENGTH = 30;
```

Comment:
Is it not possible to use dynamic truncation ? On large screens this might look weird no ?